### PR TITLE
handling unknown dates

### PR DESCRIPTION
**Pull request by anthonygarvan**
cc @jenniferthibault

### CODE REVIEW

**Comment by jenniferthibault** [comment]
Updated language, so this is ready for review 👍

File: openfecwebapp/filters.py [hunk 1]
```diff
@@ -30,6 +30,11 @@ def date_filter(value, fmt='%m/%d/%Y'):
         return None
     return ensure_date(value).strftime(fmt)
 
+@app.template_filter('ao_document_date')
+def ao_document_date(value):
+    date = date_filter(value)
+    return 'Not dated' if date == '01/01/1900' else date
```
Comment (noahmanger) [inline openfecwebapp/filters.py]: I take it the date we get from the API doesn't need any formatting?

**Comment by noahmanger** [comment]
I just had one question. Also, is there an example AO I can show to demo this?

**Comment by jenniferthibault** [comment]
Here's a handful of the most recent documents with this scenario, pulled from the CSV Tony exported and documented in https://github.com/18F/openFEC/issues/2345#issuecomment-299203220
[AO 1997-26](https://beta.fec.gov/data/legal/advisory-opinions/1997-26/), Draft Document
[AO 1998-20](https://beta.fec.gov/data/legal/advisory-opinions/1998-20/), Commissioner Statement
[AO 1998-24](https://beta.fec.gov/data/legal/advisory-opinions/1998-24/), Draft Document
[AO 1999-25](https://beta.fec.gov/data/legal/advisory-opinions/1999-25/), Draft Document
[AO 2003-13](https://beta.fec.gov/data/legal/advisory-opinions/2003-13/), Final Opinion

File: openfecwebapp/filters.py (hunk 1 above)
Comment (anthonygarvan) [inline openfecwebapp/filters.py]: Not sure what you mean. The raw data coming back from api goes through date_filter function in line 35, that formats it.

Comment (noahmanger) [inline openfecwebapp/filters.py]: Oh duh. I just glossed over that. Thanks!

**Pull request merged**